fe/fe-core/src/main/java/org/apache/doris/catalog/ColumnIdFlushDaemon.java

```diff
@@ -0,0 +1,174 @@
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.catalog;
+
+import org.apache.doris.alter.AlterLightSchChangeHelper;
+import org.apache.doris.common.Config;
+import org.apache.doris.common.util.MasterDaemon;
+
+import com.google.common.collect.Maps;
+import org.apache.doris.persist.AlterLightSchemaChangeInfo;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.ReadWriteLock;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+import java.util.function.BiConsumer;
+
+/**
+ * note(tsy): this class is temporary, make table before 1.2 to enable light schema change
+ */
+public class ColumnIdFlushDaemon extends MasterDaemon {
+
+    private static final Logger LOG = LogManager.getLogger(ColumnIdFlushDaemon.class);
+
+    /**
+     * db name -> (tbl name -> status)
+     */
+    private final Map<String, Map<String, FlushStatus>> resultCollector;
+
+    private final ReadWriteLock rwLock;
+
+    private final BiConsumer<Database, OlapTable> flushFunc;
+
+    public ColumnIdFlushDaemon() {
+        super("colum-id-flusher", TimeUnit.HOURS.toMillis(1));
+        resultCollector = Maps.newHashMap();
+        rwLock = new ReentrantReadWriteLock();
+        if (Config.enable_convert_light_weight_schema_change) {
+            flushFunc = this::doFlush;
+        } else {
+            flushFunc = (db, table) -> record(db.getFullName(), table.getName(), FlushStatus.init());
+        }
+    }
+
+    @Override
+    protected void runAfterCatalogReady() {
+        flush();
+    }
+
+    private void flush() {
+        List<Database> dbs = Env.getCurrentEnv().getInternalCatalog().getDbs();
+        rwLock.writeLock().lock();
+        try {
+            for (Database db : dbs) {
+                db.getTables()
+                        .stream()
+                        .filter(table -> table instanceof OlapTable)
+                        .map(table -> (OlapTable) table)
+                        .filter(olapTable -> !olapTable.getTableProperty().getUseSchemaLightChange())
+                        .forEach(table -> flushFunc.accept(db, table));
+            }
```

Comment:
For every db, we need sleep 3s, to avoid too often to call be and the network may be crowded.